kong/pdk/service.lua

```diff
@@ -72,9 +72,8 @@ local function new()
   -- Load-balancing components such as retries and health-checks will also be
   -- ignored for this request.
   --
-  -- The `host` argument expects a string containing the IP address of the
-  -- upstream server (IPv4/IPv6), and the `port` argument must contain a number
-  -- representing the port on which to connect to.
+  -- The `host` and `port` arguments expect the values as set on a Service
+  -- entity.
```

Comment:
```suggestion
  -- The `host` argument expects the hostname or IP address of the upstream 
  -- server, and the `port` expects a port number. The values provided must 
  -- match the values set for the related Service entity.
```
How about something like this? It might seem obvious, but it's good to call out both the exact expected value formats and where to look for them.

Reply:
Thx @lena-larionova . Not a native speaker, but; "_The values provided must match the values set for the related Service entity._" tells me that the values must be equal to the values in the Service. Yet, that is not the case; they only follow the same constraints.

Reply:
I agree with what @lena-larionova has suggested as it is pretty clear and aligns well with this PR too! However the second part of the statement is confusing. Hence lets have
```lua
-- The `host` argument expects the hostname or IP address of the upstream 
-- server, and the `port` expects a port number
```